attic/rfc_reduction.md

```diff
@@ -0,0 +1,116 @@
+
+```
+ Feature Name: `Forecasting Reduction Techniques`
+ Start Date: 2019-11-16
+ Target Date:
+ ```
+
+## Summary
+[summary]: #summary
+ 
+ - Lagged values of a timeseries can be used as features for a machine learning algorithm.
+ - Two major apporaches: rolling window regression and expanding window regression
```

Comment:
Can you point me to where you are seeing the rolling window and expanding window regression vocabulary? That's normally just a thing in cross-validation.

Reply:
Rolling window only for now